### ISSUE TITLE
Missing space after `with` on API Reference

### ISSUE THREAD

**Issue by jsilvermist** [Polymer/docs#1982]
Any API reference that has a `with` section for the class is not spaced properly, eg.
![array-selector](https://cloud.githubusercontent.com/assets/10246291/24336490/2afdd2ec-125e-11e7-8d0a-b3988733a97f.png)
![dom-bind](https://cloud.githubusercontent.com/assets/10246291/24336485/142ba3dc-125e-11e7-9b39-57e1ac3f335f.png)
![polymer-element](https://cloud.githubusercontent.com/assets/10246291/24336484/142ad3ee-125e-11e7-8747-5be8952ce7e7.png)
and others.

**Comment by arthurevans** [comment]
@justinfagnani @rictic is there already a bug for this on analyzer/iron-doc-viewer or wherever it belongs?

**Comment by rictic** [comment]
I think this was fixed with https://github.com/PolymerElements/iron-doc-viewer/commit/cf52573896b17c209c26109320fe8c4bf81152ad so maybe it'll be fixed on next redeploy?

**Comment by arthurevans** [comment]
Turns out it was _not_ fixed by that change. Looks like vulcanization and/or minification and/or some mysterious property of the `<template is="dom-repeat">` is eating the space. I patched the version on the site and put in a PR for iron-doc-viewer: https://github.com/PolymerElements/iron-doc-viewer/pull/109

At the moment CI is failing for that PR, but I don't think it's my fault.

It would be good to figure out what's causing the space to be dropped, since this may affect other people's elements, as well.
